augly/image/functional.py

```diff
@@ -1759,6 +1759,56 @@ def shuffle_pixels(
     return imutils.ret_and_save_image(aug_image, output_path)
 
 
+def skew(
+    image: Union[str, Image.Image],
+    output_path: Optional[str] = None,
+    level: float = 0.5,
+    axis: str = "x-axis",
+    metadata: Optional[List[Dict[str, Any]]] = None,
+) -> Image.Image:
+
+    """
+    Skew image with respect to X-axis or Y-axis
+
+    @param image: the path to an image or a variable of type PIL.Image.Image
+        to be augmented
+
+    @param output_path: the path in which the resulting image will be stored.
+        If None, the resulting PIL Image will still be returned
+
+    @param level: the level of skew to apply to the image; a larger value means the skew
+        will be more intense
+
+    @param axis: Axis along which the image will be skewed; can be 'x-axis' or 'y-axis'
+
+    @param metadata: if set to be a list, metadata about the function execution
+        including its name, the source & dest width, height, etc. will be appended
+        to the inputted list. If set to None, no metadata will be appended or returned
+
+    @returns: the augmented PIL Image
+    """
+
+    image = imutils.validate_and_load_image(image)
```

Comment:
```suggestion
    """
    image = imutils.validate_and_load_image(image)
```
remove extraneous newline